tests/cypress/page-object/exportDialog.ts

```diff
@@ -0,0 +1,19 @@
+import {BaseComponent, Button, Dropdown, getComponentByAttr} from '@jahia/cypress';
+
+export class ExportDialog extends BaseComponent {
+    static defaultSelector = 'button[data-sel-role="breadcrumb-item"]';
+
+    export(downloadsFolder:string, workspace:string = 'Staging content only', exportFormat: string = 'zip') {
+        const dialog = getComponentByAttr(BaseComponent, 'data-cm-role', 'export-options');
+        dialog.should('be.visible');
```

Comment:
no need to fetch dialog here as this class should be the export dialog. this selector should be changedas default selector instead.